lib/asn1/decoders/der.js

```diff
@@ -244,6 +244,10 @@ DERNode.prototype._decodeInt = function decodeInt(buffer, values) {
   var raw = buffer.raw();
   var res = new bignum(raw);
 
+  if ((raw[0] & 0x80) > 0) { // negative
```

Comment:
Should it be `!== 0`? Please consider putting the comment on the line before `if`.

Reply:
Another note: should we rather use `bignum.fromTwos` here, and avoid creating `new bignum` if the result is negative?

Reply:
As far as I can tell, bignum.fromTwos doesn't exist.